tools/python/gen_opkernel_doc.py

```diff
@@ -41,7 +40,7 @@ def format_param_strings(params):
     firstparam = True
     s = ''
     if params:
-        for param in params:
+        for param in sorted(params):
```

Comment:
sort wherever possible to avoid order ambiguity and thus avoid False Positive diffs in the md files